nomad/job_endpoint.go

```diff
@@ -114,6 +115,7 @@ func (j *Job) Register(args *structs.JobRegisterRequest, reply *structs.JobRegis
 
 				cfg, err := structs.ParseHostVolumeConfig(vol.Config)
 				if err != nil {
+					j.logger.Info("Failed to parse config for volume: %s", vol.Name)
```

Comment:
Why log this instead of returning it as the error?

Reply:
Needs a rebase - this goes away on latest master now we droped the `config` block :)